src/timesync/timesyncd-manager.h

```diff
@@ -27,6 +27,8 @@ typedef struct Manager Manager;
 #define NTP_RETRY_INTERVAL_MIN_USEC     (15 * USEC_PER_SEC)
 #define NTP_RETRY_INTERVAL_MAX_USEC     (6 * 60 * USEC_PER_SEC) /* 6 minutes */
 
+#define RETRY_USEC (30*USEC_PER_SEC)
```

Comment:
please name this DEFAULT_CONNECTION_RETRY_USEC to keep the naming in sync with the user-facing config option (of course they'll have a different sec/usec suffix, but that's fine)